backend/src/types/user_params.go

```diff
@@ -0,0 +1,11 @@
+package types
+
+type UserParams struct {
```

Comment:
is params a good name for this? or would it be better as something like [this](https://github.com/GenerateNU/sac/blob/main/backend/src/models/category.go#L13)